genie-web/src/main/resources/static/scripts/Main.js

```diff
@@ -32,7 +32,7 @@ export default class App extends React.Component {
   }
 
   componentDidMount() {
-    fetch("/actuator/info", null, "GET", "application/json").done(data => {
+    fetch("/admin/info", null, "GET", "application/json").done(data => {
```

Comment:
Commit message does not conform to best practices (partly because it's doing 2 things)

Reply:
How is this two things? It literally says `Update Actuator endpoints to /admin to match internal standard`?